redis/command_factory.hh

```diff
@@ -29,12 +29,11 @@ namespace service {
 }
 
 namespace redis {
-using namespace seastar;
 class request;
 class command_factory {
 public:
     command_factory() {}
     ~command_factory() {}
-    static future<redis_message> create_execute(service::storage_proxy&, request&, redis::redis_options&, service_permit);
+    static seastar::future<redis_message> create_execute(service::storage_proxy&, request&, redis::redis_options&, service_permit);
```

Comment:
We can choose to migrate away from this practice. In general it's good to use namespaces to have clear separation of where the library begins. Maybe we can import `future` because it's so widespread.

Reply:
Ok. Since Avi is usually a strict coding-styler, but he oked this change, I'll commit :-)